test/e2e/lib/hooks/browser.js

```diff
@@ -12,3 +19,10 @@ export const closeBrowser = async () => {
 	const driver = global.__BROWSER__;
 	await quitBrowser( driver );
 };
+
+export async function createBrowser() {
+	this.timeout( startBrowserTimeoutMS );
+	const driver = await startBrowser();
+	await ensureNotLoggedIn( driver );
+	global.__BROWSER__ = driver;
```

Comment:
The `global.__BROWSER__` assignment is already happening [inside `startBrowser`](https://github.com/Automattic/wp-calypso/blob/2e401878be47d3689db3bc53202b2725a8cd4358/test/e2e/lib/driver-manager.js#L173), so we should be OK dropping the last line here. 
```suggestion
	const driver = await startBrowser();
	await ensureNotLoggedIn( driver );
```

Reply:
Good catch!. What do you think about dropping the global assignment inside `startBrowser()` instead? I think I prefer to have it here, feels more related to how the hooks work than to the goal of `startBrowser()`

Reply:
I don't really like global variables in general as they're easy to override/mutate. Do we actually need the `__BROWSER__` to be global? Or could we simply have it enclosed in the `driverManager` module and have a getter for it?

```js
// in driver-manager.js

let driver;

function startBrowser() {
  // start the Browser instance
  driver  = new Browser().build();
}

function getBrowserDriver() {
  if ( ! driver ) {
    throw new Error( 'Browser driver is not defined. Is the browser running?' );
  }
  return driver;
}
```

This way we also have more control in case the driver is not there for some reason.